OpenSim/Simulation/Model/AbstractTool.cpp

```diff
@@ -553,22 +553,31 @@ printResults(const string &aBaseName,const string &aDir,double aDT,
 }
 
 
-bool AbstractTool::createExternalLoads( const string& aExternalLoadsFileName, Model& aModel, const Storage *loadKinematics)
+bool AbstractTool::createExternalLoads( const string& aExternalLoadsFileName,
+                                        Model& aModel, const Storage *loadKinematics)
 {
     if(aExternalLoadsFileName==""||aExternalLoadsFileName=="Unassigned") {
         cout<<"No external loads will be applied (external loads file not specified)."<<endl;
         return false;
     }
 
+    Model copyModel = aModel;
```

Comment:
I prefer `modelCopy`, used elsewhere throughout the codebase. `copyModel` sounds like an action, better suited for a function name.